test/test_images/performance/main.go

```diff
@@ -63,6 +64,8 @@ func init() {
 	// role=aggregator
 	flag.StringVar(&listenAddr, "listen-address", ":10000", "Network address the aggregator listens on.")
 	flag.UintVar(&expectRecords, "expect-records", 1, "Number of expected events records before aggregating data.")
+	flag.StringVar(&additionalTags, "additional-tags", "", "Comma separated tags to tag individual"+
```

Comment:
Suggestion to shorten the description: "Comma separated list of benchmark specific Mako tags."